webapp/app/templates/fields/multiple_input_field.html

```diff
@@ -0,0 +1,24 @@
+{{ joined_input_fields }}
+
+
+<script>
+    document.addEventListener('DOMContentLoaded', () => {
+        const sub_fields = document.querySelectorAll('input[name={{ name }}]');
+        sub_fields.forEach(input => {
+            input.addEventListener('paste', e => {
+                const data = e.clipboardData.items[0];
+                if(data.kind == 'string' && data.type == 'text/plain') {
+                    data.getAsString(str => {
+                        let start_idx = 0
+                        sub_fields.forEach(sub_field => {
+                            const length = parseInt(sub_field.dataset.fieldLength);
+                            const paste_content = str.substr(start_idx, length);
+                            sub_field.value = paste_content || '';
+                            start_idx += length;
+                        });
+                    });
+                }
```

Comment:
I don't think this works if we have spaces in the copied data? So I think we should add an additional case for this.

Reply:
I think I do not understand what you mean. For me, this works also with spaces in the copied data.

Reply:
To make sure that that the data is copied in the correct format (e.g. 24.12.1964 -> 24 12 1964 for the three fields), we will check the pattern and change the copied data correctly.

Reply:
I've changed it a little and we are now masking the paste content before we paste it into the corresponding fields.